### ISSUE TITLE
Amazon Appstore update

### ISSUE THREAD

**Issue by HommeCrabe** [nova-video-player/aos-AVP#1184]
### Description

The Amazon Appstore app has not been updated for 3 months (still 6.2.57). I'm encountering a critical issue that I hope has been fixed since. Will it be updated anytime soon or should I switch to the Google Play version? Thank you for considering this!

**Comment by courville** [comment]
Fair point about updating amazon appstore. However you could describe your issue.

**Comment by HommeCrabe** [comment]
Here is the issue I have, that wasn't there before 6.2.57. Tell me if I should post a new issue about it.

### Problem description

The app is loosing its ability to open files shared on my computer after some time not using it.

### Steps to reproduce the issue

1. Open Nova on Fire TV
2. Open a movie from a shared folder (SMB)
3. The movie plays fine
5. Switch off TV (the Fire TV will go in sleep mode)
6. Wait a few hours
7. Switch on TV
8. Try to open the movie again on Nova
9. The movie won't play as it gets stuck to the black loading loop screen (failed to open file)
10. Force stop Nova in Fire TV settings
11. Restart Nova
12. The movie plays fine again

### Expected behavior

The movie should play even after the Fire TV has been put in sleep mode.

### Model

Amazon Fire TV Stick 4K Max (1st gen)

### Application version and app store

Amazon Appstore 6.2.57
